spec/uo_cli_spec.rb

```diff
@@ -314,6 +316,24 @@ def select_measures(test_dir, measure_name_list, workflow = 'base_workflow.osw',
       expect((test_directory / 'run' / 'two_building_scenario' / '3' / 'finished.job').exist?).to be false
     end
 
+    it 'creates a system parameter file' do
+      system("#{call_cli} des_params --scenario #{test_scenario} --feature #{test_feature}")
+      expect((test_directory_elec / 'run' / 'baseline_scenario' / 'system_parameter.json').exist?).to be true
+    end
+
+    it 'creates a system parameter file with GHE properties' do
+      endsystem("#{call_cli} des_params --scenario #{test_scenario} --feature #{test_feature} --ghe")
+      expect((test_directory_elec / 'run' / 'baseline_scenario' / 'system_parameter.json').exist?).to be true
+      expect((test_directory_elec / 'run' / 'baseline_scenario' / 'ghe_dir').exist?).to be true
```

Comment:
This `ghe_dir` doesn't exist when I run this test. Does `ghe_size` need to be run first to create the folder?